libs/online-multiplayer/sim/multiplayer.ts

```diff
@@ -18,10 +18,9 @@ namespace pxsim {
 
     export interface SimulatorMultiplayerMessage extends SimulatorMessage {
         type: "multiplayer";
-        origin: "server" | "client";
+        origin?: "server" | "client";
         content: string;
-        id: number;
-        data: any;
+        id?: number;
```

Comment:
i dunno why this is showing as a change since it seems like you already made it @jwunderl ?

Reply:
weird